getting-started/blazor/your-first-chart-using-blazor.md

```diff
@@ -282,8 +243,6 @@ Now that the data is ready let's work on the styling, positioning, and giving yo
 
 Finally, prepare to render your first chart using the `BlazorFusionCharts` component. In your "App.js" file, replace your entire code with the code sample below, [Example](https://github.com/fusioncharts/blazor-fusioncharts#quick-start).
 
-> The JavaScript code to create a chart in Android and iOS is the same.
-
 ```javascript
```

Comment:
And for this example, let's use this examples. Copy the full code here from this file: https://github.com/fusioncharts/blazor-fusioncharts/blob/master/examples/demo/Pages/BarIn2D.razor